src/routes/Item.svelte

```diff
@@ -142,18 +177,18 @@
 	role="treeitem"
 	aria-selected={isSelected}
 	aria-expanded={isExpanded}
-	on:focus={() => dispatch('selected')}
-	on:keydown={(event) => handleKeyDownOnItem(event, item.id)}
-	on:click={(event) => {
+	onfocus={() => onSelected()}
```

Comment:
another note for the future that i realized that tying isSelected functionality to an item being in focus might not be the best approach bc any item can lose focus after being selected, without being unselected

(not a comment on your pr, translating my faulty code! just using this space to communicate to you some thoughts haha)